test/MyTested.AspNetCore.Mvc.ModelState.Test/BuildersTests/ModelsTests/ModelErrorTestBuilder{TModel}Test.cs

```diff
@@ -0,0 +1,252 @@
+﻿
+
+namespace MyTested.AspNetCore.Mvc.Test.BuildersTests.ModelsTests
+{
+    using System.Collections.Generic;
+    using Exceptions;
+    using Setups;
+    using Setups.Controllers;
+    using Setups.Models;
+    using Xunit;
+    public class ModelErrorTestBuilder_TModel_Test
```

Comment:
Before the class, there should be an empty line. Also, the naming does not follow the rest of the project's style.